src/bijectors/composed.jl

```diff
@@ -195,8 +195,8 @@ end
     for i = 2:N - 1
         temp = gensym(:res)
         push!(expr.args, :($temp = forward(cb.ts[$i], y)))
```

Comment:
Also replace the `forward` above (can't make a suggestion there):
```suggestion
        temp = gensym(:res_logjac)
        push!(expr.args, :(y, $temp = with_logabsdet_jacobian(cb.ts[$i], y)))
```